content/lxd/getting-started-cli.md

```diff
@@ -154,197 +154,372 @@ You can configure the following options during the initial configuration of LXD.
+#### Edit a profile
+You can either set specific configuration options or edit the full profile in YAML format.
+
+##### Set specific options for a profile
+To set an instance option for a profile, use the `lxc profile set` command. You need to specify the profile name and the key and value of the instance option:
+
+    lxc profile set <profile_name> <option_key>=<option_value> <option_key>=<option_value> ...
+
+To add and configure an instance device for your profile, use the `lxc profile device add` command. You need to specify the profile name, a device name, the device type and maybe device options (depending on the device type):
+
+    lxc profile device add <instance_name> <device_name> <device_type> <device_option_key>=<device_option_value> <device_option_key>=<device_option_value> ...
+
+To configure instance device options for an instance device that you have added to the profile already, use the `lxc profile device set` command.
+
+##### Edit the full profile
+Instead of setting each configuration option separately, you can provide all options at once in [YAML](https://en.wikipedia.org/wiki/YAML) format.
+
+Check the contents of an existing profile or instance configuration for the required markup. For example, the `default` profile could look like this:
+
+    config: {}
+    description: Default LXD profile
+    devices:
+      eth0:
+        name: eth0
+        network: lxdbr0
+        type: nic
+      root:
+        path: /
+        pool: default
+        type: disk
+    name: default
+    used_by:
+
+Instance options are provided as array under `config`. Instance devices and instance device options are provided under `devices`.
+
+To edit a profile using your standard terminal editor, enter the following command:
+
+    lxc profile edit <profile_name>
+
+Alternatively, you can create a YAML file (for example, `profile.yaml`) with the configuration and write the configuration to the profile with the following command:
+
+    lxc profile edit <profile_name> < profile.yaml
+
+#### Apply a profile to an instance
+Enter the following command to apply a profile to an instance:
+
+    lxc profile add <instance_name> <profile_name>
+
+!!! Tip
+    Check the configuration after adding the profile: `lxc config show <instance_name> -e`
+
+    You will see that your profile is now listed under `profiles`. However, the configuration options from the profile are not listed under `config`, because they are taken directly from the profile. This means that if you edit the profile, the changes are automatically applied to all instances that use the profile.
+
+You can also specify profiles when launching an instance by adding the `--profile`  (or `-p` for short) flag:
+
+    lxc launch <image> <instance_name> -p <profile> -p <profile> ...
+
+#### Remove a profile from an instance
+Enter the following command to remove a profile from an instance:
+
+    lxc profile remove <instance_name> <profile_name>
+
+## Run commands
```

Comment:
I would place this category above the configuration section.

Reply:
The idea is to group all how-to information for instances together, so "Images" should remain outside of this section.

For the order in the "Instances" section, I prefer to have it in a logical order of how you would do things, not how you would learn them (for learning, we have the online demo, and we might have tutorials in the future). And you would configure your instances before you use them. 
In the docs, we'll have separate pages for the different how-tos, which will make it easy for users to find the information they are looking for.